Branch 'master' can't be built

The latest version on master can't be built. It fails to compile `com.zsmartsystems.zigbee/com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zdo/command/ParentAnnounce.java` with such an error

```
[ERROR] /home/tom/Dev/git_repos/com.zsmartsystems.zigbee/com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zdo/command/ParentAnnounce.java:[44,18] cannot find symbol
[ERROR]   symbol:   class ParentAnnounceChildInfo
[ERROR]   location: class com.zsmartsystems.zigbee.zdo.command.ParentAnnounce
```

A look at `ParentAnnounce` reveals that it's importing `ParentAnnounceChildInfo` from the wrong package `com.zsmartsystems.zigbee.zcl.field` instead of `com.zsmartsystems.zigbee.zdo.field`. Or am I missing something?

--- a/com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zdo/command/ParentAnnounce.java
+++ b/com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/zdo/command/ParentAnnounce.java
@@ -14,7 +14,7 @@ import javax.annotation.Generated;
 
 import com.zsmartsystems.zigbee.zcl.ZclFieldDeserializer;
 import com.zsmartsystems.zigbee.zcl.ZclFieldSerializer;
-import com.zsmartsystems.zigbee.zcl.field.ParentAnnounceChildInfo;
+import com.zsmartsystems.zigbee.zdo.field.ParentAnnounceChildInfo;
 import com.zsmartsystems.zigbee.zcl.protocol.ZclDataType;
 import com.zsmartsystems.zigbee.zdo.ZdoRequest;
 

